src/oscar/apps/dashboard/catalogue/forms.py

```diff
@@ -27,17 +29,31 @@
 
 BaseCategoryForm = movenodeform_factory(
     Category,
-    fields=['name', 'slug', 'description', 'image', 'is_public'],
+    fields=['name', 'slug', 'description', 'meta_title', 'meta_description', 'image', 'is_public'],
     exclude=['ancestors_are_public'])
 
 
-class CategoryForm(BaseCategoryForm):
+class _SEOFormMixin(object):
+    def visible_fields(self):
```

Comment:
`visible_fields` has a specific meaning in Django forms, and I don't think we should override that here. Suggest we have a separate method e.g., `primary_form_fields` which returns the fields we render in the main tab, and then use that in the template.